src/interval.jl

```diff
@@ -187,6 +187,72 @@ Base.isopen(interval::AbstractInterval{T,L,R}) where {T,L,R} = L === Open && R =
 isunbounded(interval::AbstractInterval{T,L,R}) where {T,L,R} = L === Unbounded && R === Unbounded
 isbounded(interval::AbstractInterval{T,L,R}) where {T,L,R} = L !== Unbounded && R !== Unbounded
 
+# Minimum and maximum calls for Closed and Unbounded Intervals, just return first and last.
+function Base.minimum(interval::AbstractInterval{T,L,R}; increment=nothing) where {T,L,R}
+    first(interval) === nothing && return typemin(T)
+    return first(interval)
+end
```

Comment:
```suggestion
function Base.minimum(interval::AbstractInterval{T,L,R}; increment=nothing) where {T,L,R}
    return L === Unbounded ? typemin(T) : interval.first
end
```
OR use multiple dispatch to remove the conditional check:
```suggestion
Base.minimum(interval::AbstractInterval{T,Unbounded,R}; increment=nothing) where {T,R}
    return typemin(T)
end

Base.minimum(interval::AbstractInterval{T,Closed,R}; increment=nothing) where {T,R}
    return interval.first
end
```
Note: I'm using `interval.first` over `first(interval)` here to avoid the unbounded check `first` does